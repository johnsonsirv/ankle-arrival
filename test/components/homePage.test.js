/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import HomePage from '../../src/components/homePage';

describe('<HomePage /> rendering', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
