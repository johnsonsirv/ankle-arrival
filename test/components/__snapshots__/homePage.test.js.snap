// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> rendering renders correctly 1`] = `ShallowWrapper {}`;
